operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -60,21 +60,21 @@ message CNIConfig {
 
   string cniConfDir = 7;
 
-  string cniConfFileName = 8;
+  string cniAmbientConfigDir = 8;
```

Comment:
Why not append the new field to the last? This may have compatibility issues.

Reply:
thinks，It has been modified